tests/providerTest.js

```diff
@@ -314,6 +314,29 @@ describe('provider testing', () => {
             date: date
           }
         }
+      },
+      'public link share is created': (setup, parameters) => {
+        if (setup) {
+          const { username, userPassword, path, shareType, permissions, password } = parameters
```

Comment:
share type for the public link will always be the same you don't have to pass it from parameters